tests/framework/ROM/TimeSeries/ARMA/zero_filter_cluster.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/ROM/TimeSeries/ARMA.ZeroFilter</name>
+    <author>dylanjm</author>
+    <created>2022-01-11</created>
+    <classesTested>SupervisedLearning.ARMA</classesTested>
+    <description>
+      Tests the ZeroFilter functionality of the ARMA, including clustering and zero filtered target.
+    </description>
+  </TestInfo>
+
+  <RunInfo>
+    <WorkingDir>ZeroFilterCluster</WorkingDir>
+    <Sequence>load, train, meta, serialize, sample</Sequence>
+  </RunInfo>
+
+  <Files>
+    <Input name="input">Data.csv</Input>
```

Comment:
The 8760 training file might be considered too much for a regression test ... can we hack it down into a week with 2 clusters or something? or will that break the test too hard?